ParisBNF/abb/BNFabb88.xml

```diff
@@ -0,0 +1,303 @@
+                                    <ab type="pricking">Pricking is partly visible.</ab>
+                                    <ab type="ruling">Ruling is not visible.</ab>
+                                    <ab type="ruling" subtype="pattern" cert="low">1A-1A-1A1A/0-0/0-0/C</ab>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>  
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <seg type="script">18th-century script</seg> 
+                                <seg type="ink">Black, red.</seg>
+                                <desc>Careful, irregular; by a good scribe. <persName ref="PRS1071dAbbadi"/> describes the handwriting as <foreign xml:lang="fr">ʻguịlḥ moderneʼ</foreign>.
+                                </desc>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">
+                                <locus target="#2v"/>
+                                <desc><ref type="authFile" corresp="AT1042HagiographicScene">: <persName ref="PRS9848Walatta"/></ref>Black and white image does not allow any certain interpretation of the miniature.</desc>
+                            </decoNote>
+                            <decoNote xml:id="d2" type="miniature">
+                                <locus target="#3v"/>
+                                <desc><ref type="authFile" corresp="AT1042HagiographicScene"><persName ref="PRS9848Walatta"/>Walatta Ṗeṭros taming the hippopotamuses on Lake Tana.</ref></desc>
+                                    <note>Interpretation of the miniature is taken from
+                                    <bibl>
+                                        <ptr target="bm:BoscWion2010Abbadie"/>
+                                        <citedRange unit="page">90</citedRange>
+                                    </bibl>, containing a colorful reproduction of the miniature.
+                                    </note>
+                                
+                            </decoNote>
+                            <decoNote xml:id="d3" type="miniature">
+                                <locus target="#4r"/>
+                                <desc><ref type="authFile" corresp="AT1042HagiographicScene"><persName ref="PRS9848Walatta"/></ref>Black and white image does not allow any certain interpretation of the miniature.</desc>
+                            </decoNote>
+                        </decoDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="1ra"/>
+                                    <desc type="GuestText">
+                                        <title type="incomplete">Miracle of Walatta Ṗeṭros</title> 
```

Comment:
This implicates that we do need a separate ID for the Miracles as collection and possibly more IDs for single miracles